src/econf.py

```diff
@@ -267,6 +357,14 @@ def get_keys(kf: c_void_p, group: str) -> list[str]:
 
 
 def get_int_value(kf: c_void_p, group: str, key: str) -> int:
+    """
+    Evaluating an integer value for given group/key
```

Comment:
I am not sure that "Evaluating" is the precise word. I would use "Return an integer value ..."

Reply:
"Evaluating" came from the original libeconf documentation but I can change that